### PR TITLE
Falls back to default (english) files if current language not available.

### LINKED ISSUE #67: Multiple languages create debug logs in new language
ISSUE-105

Add French language and when in French, any navigation in Admin generates debug log:
[21-May-2014 16:09:13 UTC] PHP Warning: htmlspecialchars(): charset `CHARSET' not supported, assuming iso-8859-1 in /v153/admin2/includes/functions/general.php on line 39
[21-May-2014 16:09:13 UTC] PHP Warning: htmlspecialchars(): charset `CHARSET' not supported, assuming iso-8859-1 in /v153/admin2/includes/functions/general.php on line 39

### LINKED ISSUE #400: Multiple language-specific overrides needed in Admin init_languages
Specifically for init_languages.php

There is not currently any fallback to default if the lang-specific version isn't found for the many language files mentioned in init_languages.php

ie: default to a language-folder named '' if folder+file isn't found.

### PR DESCRIPTION
**Pull request by scottcwilson**
Fixes #400.
Fixes #67.

### CODE REVIEW

File: admin/includes/init_includes/init_languages.php [hunk 1]
```diff
@@ -31,9 +31,7 @@
 $template_dir = $template_query->fields ['template_dir'];
 
 // include the language translations
-if (file_exists(DIR_WS_LANGUAGES . $_SESSION ['language'] . '/locale.php')) {
-  include (DIR_WS_LANGUAGES . $_SESSION ['language'] . '/locale.php');
-}
+zen_pull_language_file(locale.php);
```
Comment (drbyte) [inline admin/includes/init_includes/init_languages.php]: Should be a string

**Comment by zcwilt** [comment]
:+1:

**Comment by drbyte** [comment]
:+1:

**Comment by ajeh** [comment]
:+1:

**Pull request merged**